sycl/include/sycl/ext/intel/esimd/detail/bfloat16_type_traits.hpp

```diff
@@ -104,6 +104,8 @@ inline std::ostream &operator<<(std::ostream &O, bfloat16 const &rhs) {
   return O;
 }
 
+template <> struct is_esimd_arithmetic_type<bfloat16, void> : std::true_type {};
```

Comment:
Is it something what became unused in process of developing the patch?
I don't see any uses of this struct.

Reply:
Right, your comment helped me to refresh that in memory. Thank you.